ios/FluentUI/Extensions/UIFontDescriptor+Extension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public extension UIFontDescriptor {
```

Comment:
These seem to be just helper functions that don't necessarily align with any FluentUI standard.
Why should FluentUI be the best place to host this code?